product_docs/docs/pgd/4/rel_notes/pgd_4.3.1_rel_notes.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.1"
+navTitle: "Version 4.3.1"
+---
+
+EDB Postgres Distributed version 4.3.1 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.14.18, 13.10.14, 14.7.0 and later.
+
```

Comment:
Do we have similar dependencies for Postgres Extended and/or community Postgres?